### PR TITLE
#24 french translation

### PR DESCRIPTION
**Pull request by albertofountain-aircury**

### CODE REVIEW

File: _data/content/projects/fr.yml [hunk 1]
```diff
@@ -5,228 +5,137 @@
...
+        technologies immersives. Le cycle de séminaires Culture Visuelle Contemporaine fait partie de l'offre publique de Mainel 
+        qui se concentre sur l'amélioration de la société à travers la solidarité et la culture comme outils de transformation, 
+        pour garantir que les personnes soient protagonistes de leur propre développement. Dans sa présentation, Elisa a montré 
+        comment les expériences immersives sont essentielles à la démocratisation de la culture, en transformant la visite passive 
+        en participation active, compréhension et une approche par les sentiments au même niveau que les connaissances partagées par 
+        les conservateurs et les institutions elles-mêmes. Les thèmes de cette discussion et les motivations derrière la programmation 
+        de Mainel montrent un profond souci du public, dont ils cherchent à enrichir et à responsabiliser la vie. 
+        Voir <a href="https://mainel.org/noticias/expografias-coloquio-cvc-27/" target="blank"> Nuevas expografías: se puede interactuar con la obra incluso sin que esté físicamente presente · Fundación Mainel </a>
```
Comment (juanjoseortega-aircury) [inline _data/content/projects/fr.yml:99]: This link should use the same class as all the links in the page called: `link`, also do it at english and spanish version, this was a mistake.

File: _data/content/projects/fr.yml [hunk 2]
```diff
@@ -5,228 +5,137 @@
 page_data:
```
Comment (juanjoseortega-aircury) [inline _data/content/projects/fr.yml:5]: After rebasing with main check the unused code, as example, lines 8 to 12 are removed, check it from the spanish or english version

File: _data/lang/fr.yml [hunk 3]
```diff
@@ -0,0 +1,263 @@
+# Mr. Green Jekyll Theme (https://github.com/MrGreensWorkshop/MrGreen-JekyllTheme)
```
Comment (juanjoseortega-aircury) [inline _data/lang/fr.yml:1]: After rebasing with main check the unused code, check it from the spanish or english version

File: fr/tabs/work.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+    <p>Quelle que soit l'étape où vous vous trouvez dans votre carrière (y compris au début) ou dans votre vie personnelle (les changements et les crises que chacun est amené à traverser sont normales et acceptées), quelle que soit votre origine ou votre localisation (nous travaillons en grande partie à distance), si vous pensez avoir quelque chose à offrir, merci d'envoyer votre CV et un portfolio si vous en avez un à :</p>
+    <button id="btn-email"><a class="link" href="mailto:jobs@curacultura.art">jobs@curacultura.art</a></button>
+    <p>Nous respectons votre temps et votre intérêt, nous répondrons donc toujours. Si nous sommes en retard, poussez-nous ! Ce sera uniquement parce que nous sommes particulièrement absorbés par un projet.</p>
+</div>
+<div>
+    <div class="photoTitle">
```
Comment (juanjoseortega-aircury) [inline fr/tabs/work.md:70]: This is also rquested by the client to be removed

**Review by juanjoseortega-aircury — CHANGES_REQUESTED** [review]
Multiple things:
1. I think that the pdf file shouldn't be uploaded here, this is a pubic repository, that means that everyone can download it.
2. This needs to be rebased with main.
3. The offering page should be removed as requested by the client.
4. Manifesto style has changed, after rebasing check the new style from english or spanish page.

File: _data/lang/fr.yml [hunk 5]
```diff
@@ -0,0 +1,167 @@
...
+  read_more_text: Lire plus 
+  worked: Nous avons travaillé avec
+  companies: Enterprises
+  museums: Musées, archives et expositions
+  festivals: Festivals
+  private_foundations: Fondations privées
+  publications: Publications
+  universities: Universités
+  selected: Selected projects
+# for contacts layout
```
Comment (cristobaldiaz-aircury) [inline _data/lang/fr.yml:88]: Missing the logo text, also in the Spanish version.

**Review by cristobaldiaz-aircury — CHANGES_REQUESTED** [review]
Just a small detail that we missed in the past.

**Pull request merged**